### PR TITLE
Housekeeping

### PR DESCRIPTION
**Pull request by yendelevium**
- Cleaned up server.go
- Cleaned up ISSUE TEMPLATES
- Cleaned up FetchData Methods

### CODE REVIEW

File: providers/meteoblue.go [hunk 1]
```diff
@@ -38,21 +39,43 @@ func NewMeteoBlueProvider() (*MeteoBlueProvider, error) {
...
-		"lat": "11.25",
-		"lon": "77",
-	}).Get("https://my.meteoblue.com/v1/forecast")
+	// Creating a new request everytime we fecth data
+	p.client.NewRequest()
+	resp, err := p.client.SetQueryParams(qp).Get(p.config.APIPath)
+	if err != nil {
+		return nil, err
+	}
+
```
Comment (adofm) [inline providers/meteoblue.go:61]: we should close the response body right ?

File: providers/open_meteo.go
Comment (adofm) [inline providers/open_meteo.go:1]: Repeating 

Over here also the body is not being closed

File: providers/meteoblue.go
Comment (JrGkOG) [inline providers/meteoblue.go:1]: the body is not being closed over here

Comment (yendelevium) [reply providers/meteoblue.go:1]: resty closes it for us automatically, we have to manually close only if its a http.Response type

Comment (JrGkOG) [reply providers/meteoblue.go:1]: okay then 
we have to manually use at certain cases of resty but now as we are not using SetDoNotParseResponce we dont need to manually close 
my bad for oversight

**Review by raamsri — APPROVED** [review]

**Pull request merged**